### DISCUSSION TITLE
preventing logs from certain dates from being deleted

### DISCUSSION THREAD

**Discussion by lravelo** [Security-Onion-Solutions/securityonion#6619] [Unsupported Versions] [answered]
we are currently conducting an internal investigation and I wanted to know if there was any way that I could prevent all indices from a certain date range from being deleted so that I don't have to recover this environment from backup.  Any info would be appreciated.  Thanks,

**Comment by weslambert** [comment, marked as answer]
You can change the value for Curator's deletion of indices temporarily while you are performing your investigation (`/opt/so/saltstack/local/pillar/global.sls`).  Otherwise, I'd advise you simply back up the time range during your investigation, or add more disk space using LVM.

**Comment by lravelo** [comment]
Thanks @weslambert. While I was going to do just this, Curator beat me too it and already deleted the logs I wanted to keep. Now I do have backups of these servers. Where should I look for these indecies and how can I reimport them? I am thinking maybe to spin up a stand-alone SO server and restore it there just so that I can use it for this purpose and not have to worry about having the data deleted.
